src/OpenLoco/Objects/ObjectManager.h

```diff
@@ -167,19 +147,14 @@ namespace OpenLoco::ObjectManager
         ObjectHeader problemObject;
     };
 
-    uint32_t getNumInstalledObjects();
-    std::vector<std::pair<uint32_t, ObjectIndexEntry>> getAvailableObjects(ObjectType type);
     void freeScenarioText();
     void getScenarioText(ObjectHeader& object);
     std::optional<LoadedObjectHandle> findIndex(const ObjectHeader& header);
```

Comment:
I suggest we rename this as it might be confused with the object index. Its finding the loaded object index from the header not be confused with finding the index from the object index.

Reply:
Good idea. How about `findIndexOffset`?